src/sentry_plugins/itunesconnect/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import
+
+from sentry_plugins.base import assert_package_not_installed
+
+assert_package_not_installed('sentry-itunesconnect')
```

Comment:
This isn't needed. This was only for legacy plugins. This package you're referencing never existed.